### PR TITLE
feat: supports modification time

### PR DESCRIPTION
**Pull request by zhenlei520**
supports modification time
IRepository adds an AnyAsync method

### CODE REVIEW

File: src/DDD/MASA.BuildingBlocks.DDD.Domain/Repositories/BaseRepository.cs [hunk 1]
```diff
@@ -23,6 +23,8 @@ public virtual async Task AddRangeAsync(IEnumerable<TEntity> entities, Cancellat
 
     public abstract Task<TEntity?> FindAsync(Expression<Func<TEntity, bool>> predicate, CancellationToken cancellationToken = default);
 
+    public  abstract Task<bool> AnyAsync(Expression<Func<TEntity, bool>> predicate, CancellationToken cancellationToken = default);
```
Comment (doddgu) [inline src/DDD/MASA.BuildingBlocks.DDD.Domain/Repositories/BaseRepository.cs:26]: This is not the responsibility of IRepository

Comment (zhenlei520) [reply src/DDD/MASA.BuildingBlocks.DDD.Domain/Repositories/BaseRepository.cs:26]: The framework itself provides basic additions, deletions, changes, and checks, and the rest are left to the user to expand?

Comment (zhenlei520) [reply src/DDD/MASA.BuildingBlocks.DDD.Domain/Repositories/BaseRepository.cs]: processed

**Review by doddgu — APPROVED** [review]
LGTM

**Pull request merged**